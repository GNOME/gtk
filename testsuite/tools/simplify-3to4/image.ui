<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkWindow">
    <child>
      <object class="GtkImage">
        <property name="pixbuf">test.svg</property>
      </object>
    </child>
  </object>
</interface>
