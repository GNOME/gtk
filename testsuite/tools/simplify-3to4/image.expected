<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk" version="4.0"/>
  <object class="GtkWindow">
    <property name="child">
      <object class="GtkImage">
        <property name="visible">0</property>
        <property name="file">test.svg</property>
      </object>
    </property>
  </object>
</interface>
